src-electron/db/db-api.js

```diff
@@ -400,6 +400,17 @@ async function insertOrReplaceSetting(db, category, key, value) {
     [category, key, value]
   )
 }
+/**
+ * Returns an array of containing the entire SETTING table
+ *
+ * @param {*} db
+ * @returns  An array containing the entire SETTING table
+ */
+async function saveSettings(db) {
+  let rows = []
+  rows = await dbAll(db, 'SELECT * FROM SETTING')
```

Comment:
Move this function into the query-<xxx>.js files. It can be used outside of db-api.js.

Also:
   the name of this function is "saveSettings". Which makes me assume it "saves" the settings.
However it does not "save" the settings, it "retrieves" or "selects" the settings.

Please name it matching to what it does.

Reply:
Okay, I am on it

Reply:
For now, the new function selectSetting() is still in db-api.js as we discussed..